query-service-impl/src/main/java/org/hypertrace/core/query/service/trino/QueryRequestToTrinoSQLConverter.java

```diff
@@ -60,10 +63,10 @@ Entry<String, Params> toSQL(
     trinoExecutionContext.clearActualTableColumnNames();
 
     paramsBuilder.addStringParam(trinoExecutionContext.getExecutionContext().getTenantId());
-    if (request.hasFilter()) {
+    Filter filter = trinoFilterHandler.skipTrinoAttributeFilter(request.getFilter());
```

Comment:
will need to check if `request.hasFilter()` inside `skipTrinoAttributeFilter`?

0r need to handle the DEFAULT_INSTANCE, right?

Reply:
Retained the existing check to be extra safe